content/en/docs/services/coap-gateway/coap-gateway.md

```diff
@@ -35,6 +35,20 @@ This authorization is tied to the connection, meaning that if a device gets disc
 
 Once device authentication is completed, the synchronization of the device twin begins. The device publishes resources, and the CoAP gateway subscribes to these resources, forwarding any changes to the IoT hub to update the device twin.
 
+### Efficient Device Twin Synchronization using Entity Tag (ETAG)
+
+The CoAP gateway employs the Entity Tag (ETAG) mechanism to monitor resource changes. An ETAG, which is a hash of a resource's content, is used to determine if a resource has been modified on the device. The Hub stores the ETAG for each resource along with the timestamp of the last change.
```

Comment:
`An ETAG, which is a hash of a resource's content` is this a different implementation than in IoT (which will use time)?

Reply:
No ETag is opaque data for the `hub`. - Removed